### DISCUSSION TITLE
How to debug such errors?

### DISCUSSION THREAD

**Discussion by studioab** [lost-pixel/lost-pixel#246] [General]
![image](https://user-images.githubusercontent.com/13108111/216865371-21fb5baa-4b74-4b8e-9900-c16755ff7d19.png)

**Comment by d-ivashchuk** [comment]
hey @studioab! this seems quite obscure, probably something is uncaught on our side! do you mind sharing more information with regards to where are you running lost-pixel e.g. OS, mode in which you are trying to run and any other information that you may find useful to share!

Thanks!

**Reply by studioab** [reply]
it only reproes with yarn workspaces, i've reinstalled project deps with npm and issue is gone.
